site/content/docs/02-template-syntax.md

```diff
@@ -124,6 +124,8 @@ An element or component can have multiple spread attributes, interspersed with r
 
 > The `value` attribute of an `input` element or its children `option` elements must not be set with spread attributes when using `bind:group` or `bind:checked`. Svelte needs to be able to see the element's `value` directly in the markup in these cases so that it can link it to the bound variable.
 
+> Sometimes, the attribute order matters as Svelte sets attributes sequentially in JavaScript. For example, `<input type="range" min="0" max="1" value={0.5} step="0.1"/>`, the value will be 1 because `step` is set after `value` is set. To fix it, `value` should set last.
```

Comment:
Just realise I forgot to submit the suggestion I wrote long ago

Reply:
What do you think?

Reply:
I think that @baseballyama's version is easier to understand for the range example as it's shorter. Adding the lazy loading image example might be good since I imagine it's a fairly common thing to write. I'm a bit conflicted on having two examples, but think it might be worth it in this case given how easy it might be to inadvertently trigger this issue

Reply:
@baseballyama your call. im just offering another example since it was mentioned in one of the issues to be closed

Reply:
@tanhauhau Sorry for super late reply.
Yeah, I read again the sentence and it's very helpful to understand. So I updated it.